stanford_courses.info

```diff
@@ -120,4 +124,5 @@ features_exclude[field_instance][field_collection_item-field_s_image_info-field_
 features_exclude[field_instance][field_collection_item-field_s_image_info-field_s_image_credits] = field_collection_item-field_s_image_info-field_s_image_credits
 features_exclude[field_instance][field_collection_item-field_s_image_info-field_s_image_image] = field_collection_item-field_s_image_info-field_s_image_image
 features_exclude[field_instance][field_collection_item-field_s_image_info-field_s_image_source_info] = field_collection_item-field_s_image_info-field_s_image_source_info
-project status url = 'https://github.com/SU-SWS/stanford_courses'
+project path = sites/all/modules/stanford
```

Comment:
When you export from the UI and have the export path field filled in with a value it does.